app/navigation/TransportUnitNavigator.scala

```diff
@@ -74,6 +75,13 @@ class TransportUnitNavigator @Inject() extends BaseNavigator {
           controllers.routes.JourneyRecoveryController.onPageLoad()
       }
 
+    case TransportUnitsReviewPage => (userAnswers: UserAnswers) =>
+      userAnswers.get(TransportUnitsReviewPage) match {
+        case Some(ChangeAnswers) => controllers.sections.transportUnit.routes.TransportUnitsAddToListController.onPageLoad(userAnswers.ern, userAnswers.arc)
```

Comment:
Although this is correct, would this actually make sense to go back through the `IndexController`? So that:
  1) You'd only have to do the SaveAndRedirect to copy the UserAnswers when they actually want to make a change _(on the Index controller)_
  2) That routing logic to the section is centralised.